.gitlab-ci.yml

```diff
@@ -4,6 +4,12 @@ stages:
 variables:
   DEBUG: "1"
 
+workflow:
+  rules:
+    - if: $CI_COMMIT_REF_NAME == "test"
+      when: always
+    - when: never
```

Comment:
manual? but also: maybe this limit can be set only for token-needing jobs?

Reply:
Manual is refused here from CI lint. I can override temlplate job and mypy, it will be easier.